client/src/components/Reviews.jsx

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+
+class Reviews extends React.Component {
+  constructor (props) {
+    super (props);
+    
+    this.state = {
+      reviews: [],
+      users: []
+    }
+
+    this.getReviews = this.getReviews.bind(this);
+
+  }
+
+  componentDidMount() {
+    this.getReviews(3)
+  }
+
+  getReviews (restaurant_id) {
+    axios.get(`/restaurant?id=${restaurant_id}`)
```

Comment:
Hi Kim,
Please look at "Route parameters" in the link below. It will make the endpoint of your HTTP request look better.
[https://expressjs.com/en/guide/routing.html](url)

Reply:
Thanks Roger! I adjusted the endpoint as your suggestions